suse/rdma.sriov-rules

```diff
@@ -0,0 +1,7 @@
+ACTION=="remove", GOTO="rdma_sriov_end"
+SUBSYSTEM!="infiniband", GOTO="rdma_sriov_end"
+
+# Automatically load general RDMA ULP modules when RDMA hardware is installed
+TAG+="systemd", ENV{ID_RDMA_INFINIBAND}=="1", ENV{SYSTEMD_WANTS}+="rdma-sriov.service"
+
```

Comment:
redhat/rdma.sriov-vfs talks about ethernet ports, so this probably needs to be a wider..

Arguably we need ID_RDMA_SRIOV as well to do this.

Reply:
I think the ethernet part is for mlx4 boards in ethernet mode which would be marked as ID_RDMA_INFINIBAND anyway (use mlx4_core driver)

Reply:
Right, but fixing that is on my hitlist too :) So it is OK for now